### PR TITLE
Add Altered ALTE to Terraswap

### PR DESCRIPTION
**Pull request by 0xantman**
Hey guys! Can you review our PR ? thank you

### CODE REVIEW

File: src/rest/usePairs.ts [hunk 1]
```diff
@@ -304,6 +304,29 @@ export default () => {
               ) {
                 return
               }
+              
+              if (
```
Comment (JoowonYun) [inline src/rest/usePairs.ts:308]: https://github.com/terra-money/assets/pull/48

If the corresponding pr are merged, there is no need for the above branching statement. We trust the whitelist of terra-money/asset.

File: src/helpers/token.ts [hunk 2]
```diff
@@ -205,6 +205,7 @@ export const GetTokenSvg = (symbol: string) => {
         case "mAMD":
         case "bETH":
           res = "/images/CW20/" + symbol + ".png"
+        case "ALTE":
```
Comment (JoowonYun) [inline src/helpers/token.ts:208]: The sentence above must apply above the sentence below.

```
case "bETH":
case "ALTE":
    res = "/images/CW20/" + symbol + ".png"
```

**Comment by JoowonYun** [comment]
Please squash unnecessary commits. 🙏

File: src/helpers/token.ts (hunk 2 above)
Comment (0xantman) [reply src/helpers/token.ts:209]: Ok fixed thank you

File: src/rest/usePairs.ts (hunk 1 above)
Comment (0xantman) [reply src/rest/usePairs.ts]: OK removed this statement for now

**Review by JoowonYun — APPROVED** [review]

**Pull request merged**